components/cp_services.php

```diff
@@ -3,18 +3,37 @@
 include_once '../includes/config.php';
 include '../includes/db_api.php';
 
+// verifica se o id do genero foi fornecido na url através do método GET
+if (isset($_GET['genre_id'])) {
+  $genre_id = $_GET['genre_id'];
 
-//// verifica se o id do genero está no URL
-if(isset($_GET['genre_id'])) {
-    $genre_id = $_GET['genre_id'];
+  $services = fetchServicesByGenreId($genre_id);
 
-    $services = fetchServicesByGenreId($genre_id);
+  
+  echo "<h2>Services for Genre: $genre_id</h2>";
+  if (count($services) > 0) {
+    echo "<ul>";
+    //loop foreach é usado para iterar sobre cada serviço
+    foreach ($services as $service) {
+      $serviceName = $service['service_name'];
+      $servicePhoto = $service['service_photo']; 
+      $serviceId = $service['id_services'];
 
-    header('Content-Type: application/json');
-    echo json_encode($services);
+      
+      $imagePath = "../imgs/dep_laser/dep_laser_" . $serviceId . ".jpg";  
+
+      echo "<li>
+      <h3><a href='cp_detail.php?service_id=$serviceId'>$serviceName</a></h3>";
+        echo "<img src='$imagePath' alt='$serviceName Service Photo'>";
+      echo "</li>";
```

Comment:
Tecnologia
Aqui podes usar a notação `[Herodoc](https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc)`
```suggestion
      echo <<<END
      <li>
          <h3><a href='cp_detail.php?service_id=$serviceId'>$serviceName</a></h3>
          <img src='$imagePath' alt='$serviceName Service Photo'>
      </li>
      END;
```

Reply:
Também podes usar noutros sítios. Este foi o mais óbvio que encontrei. Mas em qualquer sítio com múltiplos `echo` seguidos, para múltiplas linhas de HTML, podes fazer isto.